index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-size: cover;
+    }
+    </style>
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
```

Comment:
- [ ] Kindly remove the list of menus too because in this milestone you have to do only the hamburger button. 👍